server/pbench/bin/pbench-satellite-cleanup

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+import sys
+import os
+import shutil
+for line in sys.stdin:
+    tarball = line.split(": ")[1]
+    try:
+        src = "TO-SYNC/%s" % tarball
+        des = "SYNCED"
+        shutil.move(src, des)
+    except Exception as e:
+        print (e)
+        continue
+    try:
+        os.remove('%s' % tarball)
```

Comment:
Here, meaning at this place in the code? or here, meaning on the remote host? The ultimate object is to remove not only the tarball, its MD5 sum and any prefix file as is done by this script, but also the corresponding directory in the remote's "incoming" and "results" hierarchy. That will be added, once we are comfortable that the logic is correct and the implementation is bullet-proof.